installer/Migration/V5_2_0/lang-string/buzz.yaml

```diff
@@ -0,0 +1,7 @@
+langStrings:
+  -
+    value: 'Upcoming Anniversaries'
+    unitId: upcoming_anniversaries
+  -
+    value: 'Years'
+    unitId: years
```

Comment:
`years` is not defined before? please check

Reply:
No

Reply:
okay. i also noted now this word can have both singular and plural use in this component.

![image](https://user-images.githubusercontent.com/12880059/192195544-cf615d4f-f8c5-429b-b25b-3412da3a2e26.png)

Reply:
add years and year langstrings